_kb/nbt-discourse-assistantbot.md

```diff
@@ -0,0 +1,96 @@
+---
+title: NuBits Discourse Assistant Bot
+nav_heading: Tools
+filename: nbt-discourse-assistantbot.md
+permalink: /nubits-discourse-bot/
+layout: kb
+lang: en
+callouts:
+---
+NuBits Discourse Assistant Bot it is designed to be used on [NuBits official forum](https://discuss.nubits.com)  to retrieve information about NuNet (liquidity, custodians, votes...)  formatting Motions or Custodian proposals and tipping NuBits on the forum.  Just mention `@assistant` in a public discussion or send a PM to access the bot (scroll down for the list of commands).
+
+It comes as a [discourse](http://www.discourse.org/) plugin and its code is open source and can be found  [on bitbucket](https://bitbucket.org/mj2p/assistantbot). 
+
+As well as helping in the formatting of motions , the bot can also fetch any of the NuNet information available in the Nu client and present it here in any public topic. Things like the current liquidity, the current difficulty, the current state of a particular motion or the current park rates can all be accessed with a simple command.
+Because the bot is hooked into the Nu Network, it can also let any user tip NuBits to any other user on this forum. To get started with tipping, send a PM with the word 'register' to [assistant](https://discuss.nubits.com/users/assistant/activity) . If you send someone who hasn't registered a tip, I'll create them an account and let them know a tip is waiting for them.
+
+To help get started, below  there is a list of the commands the bot will respond to. Please note that the bot will only respond to 'Private Message' commands via private message. Mention commands will only work when you mention its name in a forum post and type the command as shown below.
+
+## Private Message Commands
```

Comment:
Is intentional as they are hierarchically submenus. Do you prefer to change them to flat structure?